tests/flavor_test.go

```diff
@@ -22,10 +23,11 @@ import (
 	"kubevirt.io/kubevirt/tests/util"
 )
 
-var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Flavor and Preferences", func() {
+var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-compute] Flavor and preferences", func() {
 
 	var (
 		virtClient kubecli.KubevirtClient
+		err        error
```

Comment:
In my opinion, having an error as a top level variable is not a good design. Error should be handled in the scope where it happens, and using a variable to move it to different scope seems incorrect to me.

Can you define it in the same scope as the call the returns error? Most of the time you can use `:=`.

Reply:
ACK